lib/controllers/base.js

```diff
@@ -55,18 +56,21 @@ Controller.prototype.route = function() {
 
   // NOTE: is there a better place to do this mapping?
   var pathOptions = endpoint.string;
-  if (app.name === 'restify') {
-    if (self.method === 'delete')
+  if (this.isRestify) {
+    if (self.method === 'delete') {
```

Comment:
yeah I understand, we had a similar problem in `epilogue` which I converted into this project. I just think there are other ways to determine without having to explicitly set it. I think a good idea would be to have an explicit option during initialize, but to default to using the heuristic with some of the suggested approaches I listed above